nncf/experimental/torch/nas/bootstrapNAS/search/search.py

```diff
@@ -50,6 +48,16 @@
 from nncf.experimental.torch.nas.bootstrapNAS.elasticity.multi_elasticity_handler import SubnetConfig
 from nncf.torch.nncf_network import NNCFNetwork
 
+try:
+    import matplotlib.pyplot as plt
```

Comment:
To be more consistent, it makes sense to do the same for hawq_debug.py file
could you do that in the PR?

Reply:
Honestly, I thought that the debug code should fail if the dependencies are not meet But nevertheless, I did it.